scipy/linalg/tests/test_decomp.py

```diff
@@ -2043,12 +2046,22 @@ def test_2x2(self):
         assert_array_almost_equal(h2, b)
 
 
+blas_provider = blas_version = None
+if CONFIG is not None:
+    blas_provider = CONFIG['Build Dependencies']['blas']['name']
+    blas_version = CONFIG['Build Dependencies']['blas']['version']
```

Comment:
I'll note that this is technically not correct, because the runtime and build-time BLAS are different on conda-forge and most Linux distros. However, this should be no problem here; conda-forge is already on 0.3.21 and if there's a failure elsewhere it's a good nudge to upgrade OpenBLAS.